core/src/main/java/hudson/cli/declarative/CLIRegisterer.java

```diff
@@ -279,7 +280,7 @@ protected int run() throws Exception {
                             throw new UnsupportedOperationException();
                         }
                     }));
-                } catch (ClassNotFoundException e) {
+                } catch (ClassNotFoundException | MissingResourceException e) {
```

Comment:
Yes, see commit introducing it. The `enable-job` and `disable-job` `@CLIMethod`s were moved, and if you forget to move the corresponding `shortDescription`, as I initially did, you not only break those CLI commands, you break _every other_ CLI command too.